tests/models/mamba/test_modeling_mamba.py

```diff
@@ -387,8 +387,8 @@ def check_equivalence(model, tuple_inputs, dict_inputs, additional_kwargs={}):
 
                 def recursive_check(tuple_object, dict_object):
                     if isinstance(tuple_object, MambaCache):  # MODIFIED PART START
-                        recursive_check(tuple_object.conv_states, dict_object.conv_states)
-                        recursive_check(tuple_object.ssm_states, dict_object.ssm_states)
+                        recursive_check(tuple_object.conv_state_cache, dict_object.conv_state_cache)
```

Comment:
this is a breaking change in term of naming for the cache: people who relied on ssm_states and conv_states will have breaking behavior

Reply:
changed the name back